recipes/libsvtav1/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMakeToolchain, CMakeDeps, CMake
+from conan.tools.files import copy, get, rmdir, collect_libs
+
+required_conan_version = ">=1.47.0"
+
+
+class SVTAV1Conan(ConanFile):
+    name = "libsvtav1"
+    license = "BSD-3-Clause"
+    description = "An AV1-compliant software encoder/decoder library"
+    topics = "encoder", "ffmpeg", "av1"
+    homepage = "https://gitlab.com/AOMediaCodec/SVT-AV1"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def build_requirements(self):
+        if self.settings.arch in ("x86", "x86_64"):
+            self.tool_requires("nasm/2.15.05")
+
+    def layout(self):
+        cmake_layout(self)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
```

Comment:
The destination is not required with layout being provided.... it doesn't make in CCI workflow for recipes so it's fine its missing